From 15b4f9cc9baedec10d9eb1493b35764b2e6f2ee0 Mon Sep 17 00:00:00 2001
From: Meir Levi <mlevi4@marvell.com>
Date: Thu, 24 Dec 2020 10:13:44 -0500
Subject: [PATCH 1065/1921] dts: marvell: musdk: add cma memory node

Continuous memory allocation module used to allocate memory for
MUSDK.

Signed-off-by: Meir Levi <mlevi4@marvell.com>
Change-Id: Iba005bd43351507c5daeb7adf549223184bec7da
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/42773
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Tested-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-37xx.dtsi  | 4 ++++
 arch/arm64/boot/dts/marvell/armada-ap80x.dtsi | 4 ++++
 2 files changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
index f43c43168b00..f53e4e050c6a 100644
--- a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
@@ -460,6 +460,10 @@
 				      <0x1da0000 0x20000>; /* GICV */
 				interrupts = <GIC_PPI 9 IRQ_TYPE_LEVEL_HIGH>;
 			};
+
+			musdk_cma {
+				compatible = "marvell,musdk-cma";
+			};
 		};
 
 		pcie0: pcie@d0070000 {
diff --git a/arch/arm64/boot/dts/marvell/armada-ap80x.dtsi b/arch/arm64/boot/dts/marvell/armada-ap80x.dtsi
index 5191152bd504..7c0daadc2c66 100644
--- a/arch/arm64/boot/dts/marvell/armada-ap80x.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-ap80x.dtsi
@@ -301,6 +301,10 @@
 					#thermal-sensor-cells = <1>;
 				};
 			};
+
+			musdk_cma {
+				compatible = "marvell,musdk-cma";
+			};
 		};
 	};
 
-- 
2.31.1

